tests/test_client.py

```diff
@@ -175,20 +181,22 @@ def test_upload_events(self):
             "events": [
                 {
                     "type": "case",
-                    "testName": "test1",
+                    "testPath": ["test1"],
```

Comment:
IIUC, `testPath` is consisted of array of objects like `[{ "type": "class", "name": "ClassA" }, { "type": "case", "name": "test_a" }]`. Does this work correctly with our server?

Reply:
The code itself works fine, but as you can see [here](https://github.com/launchableinc/nose-launchable/pull/26/files#diff-0d92063e88430a02df61616c5f16b148b64ac4539d9cb9b8d883d5a23351b110R157), I mocked the test path component to return a simple string. That is why the expected body is in a slightly different form. I'll make it similar to the correct one.